test/rust_src/src/keymaps-test.el

```diff
@@ -1,10 +1,24 @@
 (require 'ert)
 
+(ert-deftest keymap-make-tests ()
+  (should (equal (make-keymap) '(keymap
+                                 #^[nil nil keymap nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil])))
+  
+  (should (equal (make-keymap "menu-name") '(keymap
+                                             #^[nil nil keymap nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil] "menu-name")))
+
+  (should (keymapp (make-keymap)))
```

Comment:
Have the `keymapp` function tested in its own function. In general, each Rust function we add should have a matching test function named after it.

Reply:
Got it. I'll be sure to separate them.

Reply:
It's worth adding a test for sparse keymaps too I think. `(should (keymapp (make-sparse-keymap)))`